app/assets/javascripts/converse.js

```diff
@@ -0,0 +1,23 @@
+//= require converse.js.js
+
+// initialize converse xmpp client
+$(document).ready(function() {
+  if (app.currentUser.authenticated()) {
+    $.post("/user/auth_token", null, function(data) {
+      if (converse && data.token) {
+        converse.initialize({
+            bosh_service_url: $('script#converse').data('endpoint'),
+            auto_login: true,
+            jid: app.currentUser.get('diaspora_id'),
+            password: data.token,
+            debug: true,
+            allow_registration: false,
```

Comment:
- Expected indentation of 10 space characters but found 12.
- Identifier 'allow_registration' is not in camel case.